### PR TITLE
Fix python unit tests

### PR DESCRIPTION
**Pull request by reyoung**

### CODE REVIEW

File: python/paddle/v2/framework/op.py [hunk 1]
```diff
@@ -39,26 +41,34 @@ def __call__(self, *args, **kwargs):
...
-            tmp_index_attr.ints.extend(tmp_index)
+        op_desc = framework_pb2.OpDesc()
+
+        for input_parameter in self.__op_proto__.inputs:
+            input_arguments = kwargs.get(input_parameter.name, [])
+            if is_str(input_arguments):
+                input_arguments = [input_arguments]
+
+            if not input_parameter.duplicable and len(input_arguments) > 1:
+                raise ValueError("Input %s only accept one output, but give %d"
```
Comment (qingqing01) [inline python/paddle/v2/framework/op.py]: Input %s only accepts one input,

File: python/paddle/v2/framework/op.py [hunk 2]
```diff
@@ -39,26 +41,34 @@ def __call__(self, *args, **kwargs):
...
+                                 % (input_parameter.name, len(input_arguments)))
+
+            ipt = op_desc.inputs.add()
+            ipt.parameter = input_parameter.name
+            ipt.arguments.extend(input_arguments)
```
Comment (qingqing01) [inline python/paddle/v2/framework/op.py:57]: `out` is used below.  So `ipt` -> `in`

File: python/paddle/v2/framework/op.py [hunk 3]
```diff
@@ -39,26 +41,34 @@ def __call__(self, *args, **kwargs):
...
+
+        for output_parameter in self.__op_proto__.outputs:
+            output_arguments = kwargs.get(output_parameter.name, [])
+            if is_str(output_arguments):
+                output_arguments = [output_arguments]
+
+            if not output_parameter.duplicable and len(output_arguments) > 1:
+                raise ValueError(
+                    "Output %s only accept one output, but give %d" %
```
Comment (qingqing01) [inline python/paddle/v2/framework/op.py]: accept -> accepts

File: python/paddle/v2/framework/op.py (hunk 2 above)
Comment (reyoung) [reply python/paddle/v2/framework/op.py:57]: in is a keyword for python

File: python/paddle/v2/framework/op.py (hunk 1 above)
Comment (reyoung) [reply python/paddle/v2/framework/op.py]: Done

File: python/paddle/v2/framework/op.py (hunk 3 above)
Comment (reyoung) [reply python/paddle/v2/framework/op.py]: Done

**Pull request merged**